pkgs/servers/irc/solanum/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, bison
+, flex
+, openssl
+, sqlite
+, lksctp-tools
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
I think this `rec` here is superfluous?

Reply:
Started referencing `pname` in `src` and we will probably reference `version` once things stabilize as well.